### PR TITLE
First commit in feature1

### PR DESCRIPTION
**Pull request by shashikiran-alloli**
Feature 1 -
-- python file

### CODE REVIEW

File: py_version.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+import os
+
```
Comment (Surya-Bansal) [inline py_version.py:3]: nice

Comment (ghost) [reply py_version.py:3]: cool

**Review by shak33b — DISMISSED** [review]

File: py_version.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+import os
```
Comment (Surya-Bansal) [inline py_version.py:2]: 1st review

**Review by Surya-Bansal — DISMISSED** [review]

**Review by ShuaibShaikh15 — DISMISSED** [review]

File: py_version.py (hunk 1 above)
Comment (AnaswaraElizabethAnt) [reply py_version.py:3]: nice cool

Comment (sampada8929) [reply py_version.py:3]: nice

File: py_version.py (hunk 2 above)
Comment (ghost) [reply py_version.py:2]: not good

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by honeysangtani** [comment]
done

**Review by shak33b — APPROVED** [review]

**Review by ShuaibShaikh15 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
ok ok okkkk

**Review by AnaswaraElizabethAnt — APPROVED** [review]
testing

**Review by Surya-Bansal — APPROVED** [review]
Final Review, bol diya to bss bol diya. #kabhikhushikabhigum

**Review by Pranita28 — APPROVED** [review]
yes

**Pull request merged**